Sources/LiveViewNative/Live/LiveView.swift

```diff
@@ -245,17 +241,21 @@ public struct LiveView<
             }
         }
         // data-confirm
-        .environment(\.eventConfirmation, session.configuration.eventConfirmation ?? { message, _ in
+        .environment(\.eventConfirmation, session.configuration.eventConfirmation ?? { [weak session] message, _ in
             return await withCheckedContinuation { continuation in
-                eventConfirmationTransaction = EventConfirmationTransaction(message: message, callback: continuation.resume(returning:))
-                showEventConfirmation = true
+                if let session = session {
+                    session.eventConfirmationTransaction = EventConfirmationTransaction(message: message, callback: continuation.resume(returning:))
+                    session.showEventConfirmation = true
+                } else {
+                    continuation.resume(returning: false)
+                }
```

Comment:
This is a particularly insidious retain cycle I want to draw attention to.  `self` was captured in a way that was retained, which captured `session` with it.  There are other leaks like this which we ought to track down but that have less immediate consequences.